src/modules/NgGrid.module.ts

```diff
@@ -0,0 +1,13 @@
+import { NgModule, ModuleWithProviders } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { BrowserModule } from '@angular/platform-browser';
+import { FormsModule } from '@angular/forms';
+import { NgGrid, NgGridItem, NgGridItemConfig, NgGridItemEvent, NgGridPlaceholder } from '../main';
+
+@NgModule({
+  imports:          [ BrowserModule, CommonModule, FormsModule ],
```

Comment:
Maybe I placed some of those into the wrong type of dependency. In the app.html which is in the src directory for what I assume is testing which has ngModule binds which now require the @angular/forms module.